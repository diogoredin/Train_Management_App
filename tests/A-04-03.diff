*** tests/A-04-03.out	2017-11-13 12:05:50.983900859 +0000
--- tests/A-04-03.outhyp	2017-11-13 12:32:43.032738225 +0000
***************
*** 34,40 ****
  5 - Gestão de passageiros
  6 - Gestão de itinerários
  0 - Sair
! Escolha uma opção: Menu principal
  1 - Reiniciar
  2 - Abrir
  3 - Guardar
--- 34,40 ----
  5 - Gestão de passageiros
  6 - Gestão de itinerários
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu principal
  1 - Reiniciar
  2 - Abrir
  3 - Guardar
***************
*** 42,45 ****
  5 - Gestão de passageiros
  6 - Gestão de itinerários
  0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 42,45 ----
  5 - Gestão de passageiros
  6 - Gestão de itinerários
  0 - Sair
! Escolha uma opção: Fim de entrada de dados (R-EOF): pt.tecnico.po.io.RuntimeEOFException: Fim do fluxo de dados de entrada
