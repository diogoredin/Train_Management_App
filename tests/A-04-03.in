3
saved03
5
4
2
Jedi
0
3
0
