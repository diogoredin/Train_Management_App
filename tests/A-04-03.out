Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome do passageiro: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 